### DISCUSSION TITLE
Error while deploying on ganache

### DISCUSSION THREAD

**Discussion by txusif** [Cyfrin/foundry-full-course-cu#227] [General]
I'm running the code on wsl on  my windows 10 machine.
When I try on deploy it on ganache using the following command:
```
forge create SimpleStorage --rpc-url http://127.0.0.1:7545 --interactive
```
The following is the Error I get:
```
Error: 
error sending request for url (http://127.0.0.1:7545/): error trying to connect: tcp connect error: 
Connection refused (os error 111)
````

**Comment by Shiva-Sai-ssb** [comment]
Hello @txusif, you can find the solution to this problem in the discussion by referring to the selected answer. Personally, I chose option 2 and it successfully resolved the issue for me.

Discussion link : https://github.com/smartcontractkit/full-blockchain-solidity-course-js/discussions/34

**Reply by txusif** [reply]
Hey @Shiva-Sai-ssb, after doing the changes in server tab I was getting the following error:
```
Error: 
custom error: EIP-1559 not activated
```
I did some googling and found out that I gotta add --legacy parameter.
Here's the command:
```
forge create SimpleStorage --rpc-url http://172.17.144.1:7545 --legacy --interactive
```

Thank you! @Shiva-Sai-ssb

**Reply by Nihavent** [reply]
Thanks @txusif the --legacy flag helped with my 
custom error: EIP-1559 not activated

**Comment by shikhar229169** [comment]
Go to settings in Ganache, then go to server, set the hostname to the vEthernet (WSL),  and change network id to 1337 and then click save and restart.

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/111875681/5e9ea690-c7cc-4a57-8386-1aac10475f2c)

**Reply by txusif** [reply]
Thank you! @shikhar229169
